### ISSUE TITLE
[locale:translation:status] Error in getting .po file

### ISSUE THREAD

**Issue by novia713** [hechoendrupal/DrupalConsole#2637]
at least for me, i'm getting errors when do `locale:translation:status`
it seems not all modules throw the error, but at least core translation package errors.

i'm investigating this, anyone can confirm this bug?
i attach some sites execution of `locale:translation:status`

![seleccion_468](https://cloud.githubusercontent.com/assets/423279/18058078/04f51b40-6e13-11e6-8b59-3ebef6791d5f.png)

![seleccion_467](https://cloud.githubusercontent.com/assets/423279/18058098/1b819f14-6e13-11e6-9e60-50c0684e3f08.png)

**Comment by novia713** [comment]
this bug is fixed at 664128dcdc0251b95ee9a755fc1176c9b3e59fd0
if you have a translation updated its type will be `LOCALE_TRANSLATION_CURRENT`
this weren't contemplated in code, so i've just added the following if
```
} else if ($project_info->type == LOCALE_TRANSLATION_CURRENT) {
   $info = $this->trans('commands.locale.translation.status.messages.translation-project-updated');
}
```

**Comment by novia713** [comment]
also,
in this `if` of *LocaleTrait.php*
```
if ($local_age >= $remote_age) {
 $info = $this->trans('commands.locale.translation.status.messages.translation-project-updated');
}
```
i  firmly suggest the following `else`
```
else{
 $info = "Module able to update";
}
```
